discretize/utils/interputils.py

```diff
@@ -13,7 +13,7 @@
 except ImportError:
     print("""Interpolation will not work, use setup.py to compile the cython:
 
-        python setup.py build_ext --inplace""")
+        python setup.py install""")
```

Comment:
Quite true, thanks @jcapriot. I will instead add a warning like this in the root init (if you installed from source and didn't run setup.py, that would break)